packages/transition-backend/src/models/db/migrations/20240503162600_addUniqueScenarioServiceIdx.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2024, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+import { Knex } from 'knex';
```

Comment:
We should really have a standard header comment with a description for each migration! (This is just a general comment)